test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-1.xml

```diff
@@ -1462,6 +1465,88 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Snap dump</output>
+	</test>
+
+	<test id="Test 69-b: Make sure CustomClassLoader (not using SCC Java API) stored classes in SCC as orphan" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $currentMode$,printStats=romclass+orphan</command>
+		<output type="success" caseSensitive="yes" regex="no">Current statistics for cache</output>
+		<output type="required" caseSensitive="yes" regex="no">ORPHAN: org/openj9/test/classtests/UnsafeBootClass</output>
+		<output type="required" caseSensitive="yes" regex="no">ORPHAN: org/openj9/test/classtests/AnonClassAndUnsafeClassTest</output>
+
+		<output type="failure" caseSensitive="no" regex="no">ROMCLASS:.*UnsafeBootClass</output>
+		<output type="failure" caseSensitive="no" regex="no">ROMCLASS:.*AnonClassAndUnsafeClassTest</output>
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Java dump</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Snap dump</output>
+	</test>
+
+	<test id="Test 69-c: Make sure orpahn classes from CustomClassLoader are re-used via compareROMClassForEquality when -XX:+ShareOrphan specified" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $currentMode$ -XX:+ShareOrphan -Xtrace:print={j9bcu.270} $CP_UTILS$ $PROGRAM_ORPHANTEST$</command>
+		<output type="success" caseSensitive="yes" regex="no">ShareOrphanTest Finished</output>
+		<output type="required" caseSensitive="yes" regex="yes" javaUtilPattern="yes">compareROMClassForEquality returns 1 for class name 'org/openj9/test/classtests/UnsafeBootClass'</output>
+		<output type="required" caseSensitive="yes" regex="yes" javaUtilPattern="yes">compareROMClassForEquality returns 1 for class name 'org/openj9/test/classtests/AnonClassAndUnsafeClassTest'</output>
+
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Java dump</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Snap dump</output>
+	</test>
+
+	<test id="Test 69-d: Make sure orpahn classes from CustomClassLoader are not re-used via compareROMClassForEquality in default mode" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $currentMode$ -Xtrace:print={j9bcu.270} $CP_UTILS$ $PROGRAM_ORPHANTEST$</command>
```

Comment:
Pls add `-XX:-ShareOrphans` here, or add a second test with that option.

Reply:
New test added.